src/Domain/Fixture/FixtureCustomizer.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace Acquia\Orca\Domain\Fixture;
+
+use Acquia\Orca\Helper\Filesystem\FinderFactory;
+use Acquia\Orca\Helper\Filesystem\FixturePathHandler;
+use Acquia\Orca\Options\FixtureOptions;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Filesystem\Filesystem;
+
+/**
+ * Customizes fixtures.
+ */
+class FixtureCustomizer {
+
+  /**
+   * The Finder Factory.
+   *
+   * @var \Acquia\Orca\Helper\Filesystem\FinderFactory
+   */
+  private $finderFactory;
+
+  /**
+   * The Filesystem service.
+   *
+   * @var \Symfony\Component\Filesystem\Filesystem
+   */
+  private $filesystem;
+
+  /**
+   * The path handler service for the fixture.
+   *
+   * @var \Acquia\Orca\Helper\Filesystem\FixturePathHandler
+   */
+  private $fixturePathHandler;
+
+  /**
+   * The output variable.
+   *
+   * @var \Symfony\Component\Console\Output\OutputInterface
+   */
+  private $output;
+
+  /**
+   * Constructs an instance.
+   *
+   * @param \Acquia\Orca\Helper\Filesystem\FinderFactory $finderFactory
+   *   The finder factory.
+   * @param \Symfony\Component\Filesystem\Filesystem $filesystem
+   *   The filesystem service.
+   * @param \Acquia\Orca\Helper\Filesystem\FixturePathHandler $fixturePathHandler
+   *   The path handler for fixture.
+   * @param \Symfony\Component\Console\Output\OutputInterface $output
+   *   The output variable.
+   */
+  public function __construct(FinderFactory $finderFactory,
+  Filesystem $filesystem,
+    FixturePathHandler $fixturePathHandler,
+    OutputInterface $output) {
+    $this->finderFactory = $finderFactory;
+    $this->filesystem = $filesystem;
+    $this->fixturePathHandler = $fixturePathHandler;
+    $this->output = $output;
+  }
+
+  /**
+   * Runs all customisations.
+   */
+  public function runCustomisations(FixtureOptions $options): void {
+    $this->perzParagraphsRemoval($options);
+  }
+
+  /**
+   * Removes paragraph module tests from drupal/acquia_perz.
+   *
+   * The package drupal/acquia_perz requires the paragraphs module for running
+   * its phpunit tests. But paragraph module is only a dev-dependency of
+   * drupal/acquia_perz and thus does not get included in the fixture unless
+   * drupal/acquia-perz is the SUT. So we are removing all tests requiring
+   * paragraphs module when drupal/acquia_perz is not the SUT.
+   */
+  public function perzParagraphsRemoval(FixtureOptions $options): void {
```

Comment:
Function names should be verbs, because they're _doing_ something:

```suggestion
  public function removePerzParagraphsTests(FixtureOptions $options): void {
```